### ISSUE TITLE
Suggestion VST Volume => Jam Faders.

### ISSUE THREAD

**Issue by AndyMcProducer** [elieserdejesus/JamTaba#975]
Hiya guys, this is more a suggestion of an update to the vst plugins section.
Recently a Jamtaba user panned all what he needed to monitor to the left and made the right Live in ninjam. This led me to think if it was possible for either another popout mixer or page tab on jamtaba which would allow for cam setup and fader which would control the amount of volume from each vst plugin used.
I hope I've explained this correct. Also possibility of low and high cut for each vst vol fader too.
A Low pass or high cut would be nice on the ninjam channels too, the High pass/low cut which is already their is very usedful.

**Comment by elieserdejesus** [comment]
Hi @AndyMcProducer , If I understand correctly the first idea is allow mix the inserted plugins. In this example
![image](https://user-images.githubusercontent.com/1012741/35306121-67781732-0083-11e8-8f73-e06e6fd5400d.png)

JamTaba will show (in some place) 3 faders, one fader per inserted plugin, right?

Reaper is using an interesting solution in some plugins: the plugin window has an output fader in the right side:
![image](https://user-images.githubusercontent.com/1012741/35306294-138c093e-0084-11e8-8c77-fbc8fd2c30e2.png)

**Comment by AndyMcProducer** [comment]
Yes Eli, it will help peeps get their volumes better set and allow them to monitor before sending which hoepfully will stop some of the guitarists coming in suddenly spamming. But guy who said to me he needed to monitor is into experimental so for him he can get it right then flow it in gently. I use all vst's int he daw but some have equipment and other ways they create.

**Comment by AndyMcProducer** [comment]
If later they could have a midi learn option on them peeps could use cheap external midi controllers to live mix. The Low cut would be great too as this saves the need for extra plugins or plugin chains. But also a High Cut, from above 8k 10k and 16k, just enough to kill some of those uneeded high's when people have too much high gain. Flat monitors or low budget and even some headphones will be dull at the high end so people can push it higher sometimes so it sounds right to them.

**Comment by AndyMcProducer** [comment]
If your able to detect vsti or vst, maybe make vsti volume fader and vst dry/wet.
